src/lib.rs

```diff
@@ -7,21 +7,25 @@
 //! is the same as the number of constraints (i.e., where the constraint
 //! matrices are square). Furthermore, Marlin only supports instances where the
 //! public inputs are of size one less than a power of 2 (i.e., 2^n - 1).
-#![deny(unused_import_braces, unused_qualifications, trivial_casts)]
-#![deny(trivial_numeric_casts, private_in_public)]
-#![deny(stable_features, unreachable_pub, non_shorthand_field_patterns)]
-#![deny(unused_attributes, unused_imports, unused_mut)]
-#![deny(renamed_and_removed_lints, stable_features, unused_allocation)]
-#![deny(unused_comparisons, bare_trait_objects, unused_must_use, const_err)]
+//#![deny(unused_import_braces, unused_qualifications, trivial_casts)]
+//#![deny(trivial_numeric_casts, private_in_public)]
+//#![deny(stable_features, unreachable_pub, non_shorthand_field_patterns)]
+//#![deny(unused_attributes, unused_imports, unused_mut)]
+//#![deny(renamed_and_removed_lints, stable_features, unused_allocation)]
+//#![deny(unused_comparisons, bare_trait_objects, unused_must_use, const_err)]
```

Comment:
We should change these to `warn`s, like the main branch has

Reply:
Ops, forgot these